### PR TITLE
Rubygem mirror support

### PR DESCRIPTION
**Pull request by o-orand**
Support a rubygem mirror to avoid network issue.

### CODE REVIEW

File: scripts/binary-builder.sh [hunk 1]
```diff
@@ -8,6 +8,10 @@ ls spec/**/*_spec.rb | split -l $(expr `ls spec/**/*_spec.rb | wc -l` / $TOTAL_G
 specs=$(cat $(ls x* | sed -n "$CURRENT_GROUP p"))
 
 gem install bundler --no-ri --no-rdoc
+if [ "${RUBYGEM_MIRROR}" != "none" ]
```
Comment (RochesterinNYC) [inline scripts/binary-builder.sh]: These lines/if checking that checks for a value of `RUBYGEM_MIRROR` are actually not necessary.

If no `RUBYGEM_MIRROR` is specified, the command executed will be `bundle config mirror.https://rubygems.org`, which will simply return the local or global bundle config value for that mirror. Ex:

```bash
$ bundle config --local mirror.https://rubygems.org
Settings for `mirror.https://rubygems.org` in order of priority. The top value will be used
Set for the current user (/Users/pivotal/.bundle/config): "https://rubygems.org"
```

**Comment by RochesterinNYC** [comment]
Once those `RUBYGEM_MIRROR` if conditionals are removed, this PR looks good to me. :+1:

**Comment by o-orand** [comment]
@RochesterinNYC Thanks for your comments. Done and I've also added mirror support to update-compile-extensions.

**Comment by RochesterinNYC** [comment]
Thanks!

**Pull request merged**